tests/foreman/cli/test_satellitesync.py

```diff
@@ -2059,3 +2059,151 @@ def test_positive_install_package_from_imported_repos(self):
+    lce_label=None,
+    cv_label=None,
+):
+    """Configures Downstream organization to sync from particular Upstream organization.
+
+    :param dwn_sat: Downstream Satellite instance.
+    :param up_sat: Upstream Satellite instance.
+    :param dwn_org: Downstream organization to be configured.
+    :param up_org: Upstream organization to sync CDN content from, default: Default_Organization
+    :param username: Username for authentication, default: admin username from settings.
+    :param password: Password for authentication, default: admin password from settings.
+    :param lce_label: Upstream Lifecycle Environment, default: Library
+    :param cv_label: Upstream Content View Label, default: Default_Organization_View.
+    :return: True if succeeded.
+    """
+    # Create Content Credentials with Upstream Satellite's katello-server-ca.crt.
+    crt_file = f'{up_sat.hostname}.crt'
+    dwn_sat.execute(f'curl -o {crt_file} http://{up_sat.hostname}/pub/katello-server-ca.crt')
+    cc = dwn_sat.cli.ContentCredential.create(
+        {
+            'name': up_sat.hostname,
+            'organization-id': dwn_org.id,
+            'path': crt_file,
+            'content-type': 'cert',
+        }
+    )
+    # Set the CDN configuration to Network Sync.
+    res = dwn_sat.cli.Org.configure_cdn(
+        {
+            'id': dwn_org.id,
+            'type': 'network_sync',
+            'url': f'https://{up_sat.hostname}/',
+            'username': username,
+            'password': password,
+            'upstream-organization-label': up_org.label,
+            'upstream-lifecycle-environment-label': lce_label,
+            'upstream-content-view-label': cv_label,
+            'ssl-ca-credential-id': cc['id'],
+        }
+    )
+    return 'Updated CDN configuration' in res
+
+
+class TestNetworkSync:
+    """Implements Network Sync scenarios."""
+
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
+        'function_synced_rhel_repo',
+        ['rhae2'],
+        indirect=True,
+    )
+    def test_positive_network_sync_rh_repo(
+        self,
+        target_sat,
+        function_sca_manifest_org,
+        function_synced_rhel_repo,
+        module_downstream_sat,
+        function_downstream_org,
+    ):
+        """Sync a RH repo from Upstream to Downstream Satellite
+
+        :id: fdb58c18-0a64-418b-990d-2233381fee8f
+
+        :parametrized: yes
+
+        :setup:
+            1. Enabled and synced RH yum repository at Upstream Sat.
+            2. Organization with manifest at Downstream Sat.
+
+        :steps:
+            1. Set the Downstream org to sync from Upstream org.
+            2. Enable and sync the repository from Upstream to Downstream.
+
+        :expectedresults:
+            1. Repository can be enabled and synced.
+
+        :CaseLevel: System
+
+        :BZ: 2213128
+
+        :customerscenario: true
+
+        """
+        assert _set_downstream_org(
+            module_downstream_sat,
+            target_sat,
```

Comment:
Just for better readability, would suggest using upstream_sat name instead of target_sat here

Reply:
I can use named parameters for better readability.